.gitignore

```diff
@@ -6,3 +6,7 @@ __pycache__
 backup
 data
 docker-compose.override.yml
+images
```

Comment:
ok, looks reasonable if we have more defaults we might just create images/defaults or something and set this to non-ignore